app/services/articles/updater.rb

```diff
@@ -38,7 +39,8 @@ def call
       send_notification = article.published && article.saved_change_to_published_at.present?
       Notification.send_to_followers(article, "Published") if send_notification
 
-      dispatch_event(article)
+      # don't send only if article keeps being unpublished
+      dispatch_event(article) if article.published || was_published
```

Comment:
Now that we moved `return unless article.published?` inside the `dispatch_event`, would this still trigger for the `was_published` case?

Reply:
`Updater` has its own `dispatch_event`.

Reply:
Maybe the `Creator` one should have been kept the same for consistensy :)